lib/ball.js

```diff
@@ -0,0 +1,61 @@
+function Ball(options, canvas){
+  this.canvas = canvas;
+  this.ctx = this.canvas.getContext('2d');
```

Comment:
I don't think every instance needs it's own context, right? We could just pass the context in when we go to draw.